server/odc-service/src/main/java/com/oceanbase/odc/service/db/schema/model/OdcDBObject.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.db.schema.model;
+
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonProperty.Access;
+import com.oceanbase.odc.metadb.dbobject.DBObjectEntity;
+import com.oceanbase.odc.service.connection.database.model.Database;
+import com.oceanbase.tools.dbbrowser.model.DBObjectType;
+
+import lombok.Data;
+
+/**
+ * @author gaoda.xy
+ * @date 2024/3/29 11:49
+ */
+@Data
+public class OdcDBObject {
```

Comment:
how about extending DBObjectIdentity?

Reply:
Good idea! But these two object has no clear lines of succession. So, not extend may be more simple?